client/src/components/AuthHeader/useStyles.ts

```diff
@@ -8,22 +8,17 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   accAside: {
     fontSize: 14,
-    color: '#b0b0b0',
+    color: '#000000',
     fontWeight: 400,
     textAlign: 'center',
     whiteSpace: 'nowrap',
     display: 'flex',
     alignItems: 'center',
     padding: '1rem',
+    paddingRight: '0.5em',
   },
   accBtn: {
-    width: 170,
-    height: 54,
-    borderRadius: theme.shape.borderRadius,
-    filter: 'drop-shadow(0px 2px 6px rgba(74,106,149,0.2))',
-    backgroundColor: '#ffffff',
-    color: '#3a8dff',
-    boxShadow: 'none',
+    color: ' #f14140',
```

Comment:
I would recommend setting colors within MUI's theme and using it to apply colors - to ensure a consistent theme across the app :)

Ditto where applicable

Reply:
Done the changes